app/assets/stylesheets/layout_components/reveal-modal.scss

```diff
@@ -20,6 +20,26 @@ dialog .close-reveal-modal {
     font-weight: normal;
   }
 
+  &--chat {
+    @extend .reveal-modal;
```

Comment:
It might be worth adding a `outline: none` either to this or to the :focus state to prevent the blue outline from showing on mobile when the section is clicked: 
<img width="469" alt="screen shot 2018-03-08 at 3 35 22 pm" src="https://user-images.githubusercontent.com/27693833/37180461-5809dbc0-22e6-11e8-9d14-e59737a89078.png">